Modules/faulthandler.c

```diff
@@ -492,14 +597,15 @@ faulthandler_enable(void)
 static PyObject*
 faulthandler_py_enable(PyObject *self, PyObject *args, PyObject *kwargs)
 {
-    static char *kwlist[] = {"file", "all_threads", NULL};
+    static char *kwlist[] = {"file", "all_threads", "c_stack", NULL};
     PyObject *file = NULL;
     int all_threads = 1;
     int fd;
+    int c_stack = 1;
```

Comment:
I don't think it should be defaulted to `True`, especially when only available on a subset of systems.

Reply:
Eh, I don't think it matters. People don't use `faulthandler` for compatibility reasons, they just enable it for debugging. I feel like not enabling the C stack trace by default will just add an extra, annoying step for users.

Reply:
I'm also in favor of keeping the stack trace.

> especially when only available on a subset of systems

Well.. I'd say it's available on many popular systems except FreeBSD/CentoS. Ubuntu and Fedora being supported are engouh IMO, at least for many users.

Is it available on macOS by the way?

Reply:
> Is it available on macOS by the way?

Probably, but I haven't checked.